doc/faq.md

```diff
@@ -0,0 +1,24 @@
+- [Can I use dragonfly in production?](#can-i-use-dragonfly-in-production)
+- [Dragonfly provides vertical scale, but we can achieve similar throughput with X nodes in a Redis cluster.](#dragonfly-provides-vertical-scale-but-we-can-achieve-similar-throughput-with-x-nodes-in-a-redis-cluster.)
+- [I get it, but why not change Redis open source?](#i_get_it_but_why_not_change_redis_open_source)
+- [What if I have some commands that are missing?](#what_if_i_have_some_commands_that_are_missing)
+
+
+## What is the license model of Dragonfly? Is it an open source?
+Dragonfly is released under [BSL 1.1](../LICENSE.md) (Business Source License). We believe that a BSL license is more permissive than AGPL-like licenses. In general terms, it means that dragonfly's code is free to use and free to change as long as you do not sell services directly related to dragonfly or in-memory datastores.
+We followed the trend of other technological companies like Elastic, Redis, MongoDB, Cockroach labs, Redpanda Data to protect our rights to provide service and support for the software we are building. 
+
+## Can I use dragonfly in production?
+License wise you are free to use dragonfly in your production as long as you do not provide dragonfly as a managed service. From a code maturity point of view, Dragonfly's code is covered with unit testing. However as with any new software there are use cases that are hard to test and predict. We advise you to run your own particular use case on dragonfly for a few days before considering production usage. 
+
+## Dragonfly provides vertical scale, but we can achieve similar throughput with X nodes in a Redis cluster.
+Dragonfly utilizes the underlying hardware in an optimal way. Meaning it can run on small 8GB instances and scale verticly to large 768GB machines with 64 cores. This versatility allows to drastically reduce complexity of running cluster workloads to a single node saving hardware resources and costs. More importantly, it reduces the complexity (total cost of ownership) of handling the multi-node cluster. In addition, Redis cluster-mode imposes some limitations on multi-key and transactinal operations while Dragonfly provides the same semantics as single node Redis.
+
+## I get it, but why not change Redis open source?
+It's impossible to reuse Redis OSS to make it multi-threaded besides what KeyDB did. With multi-threading you need to think about all things holistically. How you handle backpressure? how you do snapshotting? How you implement multi-key operations or blocking transactions? For that you need special algorithms to provide atomicity, you need fibers/coroutines to be able to block your calling context yet unblock the cpu for other tasks etc. All this was designed bottom up from scratch. BTW, Dragonfly does use some Redis code around single value data-structures like zset, hset etc...
```

Comment:
`t's impossible to reuse Redis OSS to make it multi-threaded` - too opinionated. 
I will use material from here but I prefer to write this paragraph myself.